implementations/rust/ockam/ockam_api/src/authenticator/storage/authority_enrollment_token_repository.rs

```diff
@@ -0,0 +1,20 @@
+use crate::authenticator::one_time_code::OneTimeCode;
+use crate::authenticator::Token;
+use ockam::identity::TimestampInSeconds;
+use ockam_core::async_trait;
+use ockam_core::compat::boxed::Box;
+use ockam_core::Result;
+
+/// This repository stores enrollment tokens on the Authority node
+#[async_trait]
+pub trait AuthorityEnrollmentTokenRepository: Send + Sync + 'static {
```

Comment:
I would remove `Authority` in the name.

Reply:
I think the problem is that we create the same set of tables on each machine, therefore, I wanted to emphasize that this table is only to be used for the Authority node.

Reply:
This is why there is the `SqlxDatabase::create_with_migration` function. To be able to start an instance of a database with a different set of tables. If we made it a list of `Migration`s instead of an `Option<Migration>`, we could share some table definitions between a normal node and an authority node + have some authority-specific tables.